pkg/proxy/request.go

```diff
@@ -159,3 +215,48 @@ func fileToString(f io.ReadCloser) (string, error) {
 
 	return s, nil
 }
+
+func getTags(r *http.Request) (apisLoadTestV1.LoadTestTags, error) {
+	return parseTags(r.FormValue(tags))
+}
+
+func parseTags(tagsStr string) (apisLoadTestV1.LoadTestTags, error) {
```

Comment:
Would it make sense to make this func part of `apisLoadTestV1` as `LoadTestTagsFromString` for example?